Sources_v3/APIClient/Endpoints/ChannelEndpoints.swift

```diff
@@ -128,9 +128,9 @@ extension Endpoint {
         )
     }
     
-    static func sendEvent<ExtraData: ExtraDataTypes>(cid: ChannelId, eventType: EventType) -> Endpoint<EventPayload<ExtraData>> {
+    static func sendEvent(cid: ChannelId, eventType: EventType) -> Endpoint<EmptyResponse> {
         .init(
-            path: "channels/\(cid.type)/\(cid.id)",
+            path: "channels/\(cid.type)/\(cid.id)/event",
```

Comment:
@buh I was surprised that you changed this but you didn't have to change the test. Then I realized then you didn't implement it in the first place, not event after the review :) https://github.com/GetStream/stream-chat-swift/pull/463#discussion_r484265095

Reply:
My fault. I'll add.